content/blogs/gui-setup-part1-setupapi.md

```diff
@@ -0,0 +1,104 @@
+---
+title:      "1st-stage GUI setup - September 2023: partly Wine-syncing setupapi"
+author:     "hbelusca"
+date:       2023-10-30
+tags:       [ newsletter, gui-setup ]
+---
+
+Greetings to all ReactOS followers!
+As many of you certainly are aware by now, [I have been officially hired](/project-news/hermes-belusca-hired-full-time) by the ReactOS Deutschland e.V. to develop the graphical version of the 1st-stage ReactOS installer ("1st-stage GUI setup").
```

Comment:
Question for maybe @ColinFinck or others:
is it OK to say "hired by the ReactOS Deutschland e.V." ? or maybe simpler, "hired by the ReactOS project" ?